modules/airflow/module.tf

```diff
@@ -18,15 +18,14 @@ resource "google_storage_bucket_iam_binding" "binding" {
   ]
 }
 
+//wait = false -> https://github.com/hashicorp/terraform-provider-helm/issues/683
 resource "helm_release" "kube-airflow" {
   name = "airflow-stable"
   repository = "https://airflow-helm.github.io/charts"
   chart = "airflow"
   version = "8.0.9"
   namespace = "default"
-  wait = true
-  atomic = true
-  timeout = 3600
+  wait = false

```

Comment:
dla_wait = true_ nie uruchamiają się post-install joby np. [ten](https://github.com/airflow-helm/charts/blob/main/charts/airflow/templates/jobs/job-upgrade-db.yaml) -> i wynikiem jest zapętlenie init containera do timeoutu:
kubectl logs <web-pod> -c wait-for-db-migrations